request_access/github.py

```diff
@@ -37,7 +37,15 @@ def list_teams():
             response.text,
         )
 
-    return response.json()
+    teams = response.json()
+    for team in teams:
+        members = requests.get(
+            "https://api.github.com/orgs/OpenTTD/teams/{}/members".format(team["slug"]), headers=headers
```

Comment:
I know you don't like f-strings, but given the rest is using f-strings .. maybe use it here too? Replace `"` with `'` and you can use `{team['slug']}` just fine :)

Reply:
Boo, f f-strings :)